src/app/questionGenerator.js

```diff
@@ -0,0 +1,44 @@
+class QuestionGenerator {
+
```

Comment:
After installing packages for linting, the file should be linted we the linting styles of your choice (airbnb or any other you choose)